document/core/valid/instructions.rst

```diff
@@ -389,6 +646,48 @@ Memory Instructions
    }
 
 
+.. _valid-load-extend:
+
+:math:`\K{v128.}\LOAD{M}\K{x}N\_\sx~\memarg`
+...............................................
+
+* The memory :math:`C.\CMEMS[0]` must be defined in the context.
+
+* The alignment :math:`2^{\memarg.\ALIGN}` must not be larger than :math:`8`.
+
+* Then the instruction is valid with type :math:`[\I32] \to [\V128]`.
+
+.. math::
+   \frac{
+     C.\CMEMS[0] = \memtype
+     \qquad
+     2^{\memarg.\ALIGN} \leq 8
```

Comment:
You could make this rule a bit more symmetric with others by saying `\leq M \cdot N / 8`.

Reply:
It will be `\leq N \cdot M /8`, since `N` in this case is the number of lanes, and `M` is the size of each lane. I can swap M and N in the instruction, then it will work. Wdyt?

Reply:
I don't mind switching, but isn't A*B/C the same as B*A/C?

Reply:
Sorry, I left out the parentheses there, what i meant was: `\leq N \cdot (m/8)`.

Reply:
I swapped the N and M, so it is LoadNxM, it looks a bit more symmetric as N has to do with the size of the type. I also used `N/8 \cdot M`.